Test/Web/FunctionalTests/TestApps/WebAppFW45/Web.config

```diff
@@ -32,6 +32,8 @@
       <add name="ExtensionlessUrlHandler-Integrated-4.0" path="*." verb="*" type="System.Web.Handlers.TransferRequestHandler" preCondition="integratedMode,runtimeVersionv4.0" />
     </handlers>
     <modules runAllManagedModulesForAllRequests="true">
+      <add name="DiagnosticsHttpModule" type="Microsoft.AspNet.Diagnostics.DiagnosticsHttpModule, Microsoft.AspNet.Diagnostics, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" preCondition="integratedMode,managedHandler" />
+      <add name="DiagnosticsHttpModule2" type="Microsoft.AspNet.Diagnostics.DiagnosticsHttpModule, Microsoft.AspNet.Diagnostics, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" preCondition="integratedMode,managedHandler" />
```

Comment:
This is a very good questions.
The next two lines in this app are
```
       <add name="ApplicationInsightsHttpModule" type="Microsoft.ApplicationInsights.Web.ApplicationInsightsHttpModule, Microsoft.AI.Web" preCondition="managedHandler" />
        <add name="ApplicationInsightsHttpModule2" type="Microsoft.ApplicationInsights.Web.ApplicationInsightsHttpModule, Microsoft.AI.Web" preCondition="managedHandler" />		        <add name="ApplicationInsightsHttpModule2" type="Microsoft.ApplicationInsights.Web.ApplicationInsightsHttpModule, Microsoft.AI.Web" preCondition="managedHandler" />
```

So AI module is added two times. I assume this test checks besides other things, that even if there are 2 modules, 1 request telemetry will be reported anyway. It's questionable whether AI needs to handle this properly, but anyway. 

But I'm fine with leaving just one DiagnosticHttpModule (BTW it will be renamed to TelemetryCorrelationModule)